src/main/kotlin/translations/intentions/ConvertToTranslationIntention.kt

```diff
@@ -73,12 +77,14 @@ class ConvertToTranslationIntention : PsiElementBaseIntentionAction() {
             val key = result.first ?: return
             val replaceLiteral = result.second
             try {
-                TranslationFiles.add(element, key, value)
+                if (existingKey == null || existingKey != key) {
```

Comment:
No need for the null check, `==` is null-safe.

Reply:
[1ab56e9](https://github.com/minecraft-dev/MinecraftDev/pull/2292/commits/1ab56e9784fd3512976b4f0963019ada86ac9f3a)